src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentListItemGroup.jsx

```diff
@@ -214,19 +213,7 @@ export default function AppointmentListItemGroup({ data }) {
                 'vads-u-border-color--gray-medium': isBorderBottom,
               })}
             >
-              <div style={styles.canceled}>
-                <i
-                  aria-hidden="true"
-                  className={classNames('fas', 'vads-u-margin-right--1', {
-                    'fa-phone': isVAPhoneAppointment(appointment),
-                    'fa-video': isVideo,
-                    'fa-building': isInPersonVAAppointment() || isCommunityCare,
-                    'fa-blank': isCommunityCare,
-                  })}
-                />
```

Comment:
I think this is an orphaned unused file but needs confirmation. Removing font awesome icon.